WebApiCoreSeed.Domain/Pagination/PaginationParameters.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace WebApiCoreSeed.Domain.Pagination
+{
+    using System;
+
+    /// <summary> Wrapper for common pagination parameters </summary>
+    public class PaginationParameters
+    {
+        private const int DefaultPageSize = 15;
+        private const int MaxPageSize = 100;
```

Comment:
Why using a max page size? This is to avoid a client attempting to obtain lot of records and hang out the server?